contents/tutorials/funnels.md

```diff
@@ -1,161 +1,195 @@
+* Percentage of users paid for a subscription once the free trial ended
+* Conversion rate from your cheaper plan to your more expensive plan
+* Percentage of users that actually used the product after signing up
+* Number of active users that tried the newest beta feature
 
-## Prerequisites
+Funnels are also useful for interrogating complicated sign up processes, or a features that require extensive setup. 
 
-To follow this tutorial along, you need to:
+Setting up a funnel can help you figure out _exactly_ where users drop out of the process, so you can improve that step to make sure that users complete the funnel, even if it is a "long" one.
 
-1. Have [deployed PostHog](/docs/deployment).
-1. Have started receiving events via our [snippet](/docs/integrate/client/js), one of our [integrations](/docs/integrate/overview), or our [API](/docs/api/overview)
+## Step 1: Create your funnel
 
-> **Note:** It can also be useful to have some [actions](/docs/tutorials/toolbar#creating-actions) set up, but not necessary.
+Funnels are part Insights in PostHog. 
 
-## When you should use Funnels
+Click on the '+' icon next to 'Insights' in the sidebar to start creating a new insight, or click on 'New insight' on the Insights page.
 
-Funnels are great for when your users should perform actions in succession. That is, if you need your user to do something _and then_ do something else, funnels are a great way to gather insights into that process.
+Then, in the insight builder, click on 'Funnels' to start creating your funnel.
 
-Funnels can have as many steps as you like and offer a good look into both what percentage of **unique users** are converting from one step to another, as well as identifying which users convert, and which don't.
+![Funnels insight](../images/tutorials/funnels/funnel-preview.png)
 
-A standard but very useful funnel is looking at the sign up process from end to end, for example. You can set up a funnel starting from the moment the user lands on your page to when they actually complete the process. You are then able to find out where users are dropping off, and work on improving those areas.
+The Funnels builder is split into two panes: 'Query Steps' on the left for defining your funnel, and a preview pane on the right. Changes are calculated on-the-fly, so you can see if your funnel is set up correctly as you go.
 
-## Creating your first funnel
+There's no limit to the number of steps you can add to your funnel, but let's start with a simple example first.
 
-### Step 1: Navigate to 'Funnels'
+Click on the first step to edit it and search for your desired Event or Action. In this case, we're going to select the 'Pageview' event and then apply a filter for users visiting the PostHog homepage.
 
-Our 'Funnels' feature is located in 'Insights'. After navigating to 'Insights' on the sidebar, click on the 'Funnels' tab to get to this page:
+![Building funnel video 1](../images/tutorials/funnels/funnel-step-1.mov)
```

Comment:
This isn't working for me when previewing in yarn. I think because it's a .mov and this is an image insert. 

Solution: Change this .mov to a .gif if it's short enough, or host it on YouTube and post there.

Reply:
Yeah, I know. Sorting YT access. Gifs are shit